app/src/androidTest/java/com/android/sample/screen/RecipeOverviewTest.kt

```diff
@@ -55,18 +59,25 @@ class RecipeOverviewTest {
   private lateinit var mockNavigationActions: NavigationActions
   private lateinit var mockRepository: RecipesRepository
   private lateinit var recipesViewModel: RecipesViewModel
+  private lateinit var userViewModel: UserViewModel
+
+  // used to have one ingredient in the fridge that shares a category with the recipe
+  private val sharedCategory: String = "Apple"
 
   private val mockedRecipesList =
       listOf(
           testRecipes[0].copy(
               ingredientsAndMeasurements =
                   listOf(
-                      Pair("Ingredient 1", "Ingredient 1x"),
+                      Pair(sharedCategory, "Ingredient 1x"),
```

Comment:
Hey so if I understood correctly, you made this a `String` variable this way you can put it in the `Recipe` and in the Fridge, am I right?

Reply:
Yes!